datadog_checks_dev/datadog_checks/dev/tooling/jira.py

```diff
@@ -0,0 +1,97 @@
+# (C) Datadog, Inc. 2018-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+class JiraClient:
+    API_URL = 'https://datadoghq.atlassian.net/rest/api'
+    CREATE_ENDPOINT = API_URL + '/3/issue'
+
+    def __init__(self, config):
+        self.auth = HTTPBasicAuth(config['jira']['user'] or None, config['jira']['token'] or None)
```

Comment:
Can we print a message if `user` or `token` is not set? When I tested, the error seemed more like a connection issue than a configuration one.
```
Attempt 1 of 3: An error has occurred, retrying in 2 seconds...
Attempt 2 of 3: An error has occurred, retrying in 2 seconds...
Error: Expecting value: line 1 column 1 (char 0)
```

Reply:
Sure! Do you think it would be better to include the check in the Jira client itself or the `testable.py` script?

Reply:
We can check for it in the client before we create `self.auth`.

Reply:
Yeah that sounds good!